samcli/commands/build/command.py

```diff
@@ -120,6 +120,13 @@
     "an image URI for all functions, the default SAM CLI build images will be used.",
     cls=ContainerOptions,
 )
+@click.option(
+    "--exclude",
+    "-x",
+    default=None,
```

Comment:
While it would ease some of the code, having it default to `None` makes it consistent with other settings like `--build-image` and `--container-env-var`